app/src/main/res/raw/data.json

```diff
@@ -1738,5 +1738,2365 @@
+        "topics": ["тело"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "thumb",
+        "topics": ["body"]
+        }},
+    {"RU": {
+        "word": "большой палец",
+        "topics": ["тело"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "elbow",
+        "topics": ["body"]
+        }},
+    {"RU": {
+        "word": "локоть",
+        "topics": ["тело"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "finger",
+        "topics": ["body"]
+        }},
+    {"RU": {
+        "word": "палец",
+        "topics": ["тело"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "foot",
+        "topics": ["body"]
+        }},
+    {"RU": {
+        "word": "нога",
+        "topics": ["тело"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "ankle",
+        "topics": ["body"]
+        }},
+    {"RU": {
+        "word": "лодыжка",
+        "topics": ["тело"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "hair",
+        "topics": ["body"]
+        }},
+    {"RU": {
+        "word": "волосы",
+        "topics": ["тело"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "neck",
+        "topics": ["body"]
+        }},
+    {"RU": {
+        "word": "шея",
+        "topics": ["тело"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "hand",
+        "topics": ["body"]
+        }},
+    {"RU": {
+        "word": "рука",
+        "topics": ["тело"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "wrist",
+        "topics": ["body"]
+        }},
+    {"RU": {
+        "word": "запястье",
+        "topics": ["тело"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "hip",
+        "topics": ["body"]
+        }},
+    {"RU": {
+        "word": "бедро",
+        "topics": ["тело"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "chin",
+        "topics": ["body"]
+        }},
+    {"RU": {
+        "word": "подбородок",
+        "topics": ["тело"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "knee",
+        "topics": ["body"]
+        }},
+    {"RU": {
+        "word": "колено",
+        "topics": ["тело"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "head",
+        "topics": ["body"]
+        }},
+    {"RU": {
+        "word": "голова",
+        "topics": ["тело"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "lip",
+        "topics": ["body"]
+        }},
+    {"RU": {
+        "word": "губа",
+        "topics": ["тело"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "mouth",
+        "topics": ["body"]
+        }},
+    {"RU": {
+        "word": "рот",
+        "topics": ["тело"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "nouse",
+        "topics": ["body"]
+        }},
+    {"RU": {
+        "word": "нос",
+        "topics": ["тело"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "thigh",
+        "topics": ["body"]
+        }},
+    {"RU": {
+        "word": "бедро",
```

Comment:
what is the difference with hip?

Reply:
https://hinative.com/en-US/questions/1485419